libs/service-portal/licenses/src/utils/dataMapper.ts

```diff
@@ -430,3 +433,100 @@ export const mapCategory = (id: string): Category => {
       }
   }
 }
+
+export const getLicenseDetailHeading = (type: GenericLicenseType) => {
```

Comment:
Add a task to move this to the service instead when we've got a "server side" translation

Reply:
✅